pom.xml

```diff
@@ -4,7 +4,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>4.69</version>
+    <version>4.73</version>
```

Comment:
Unrelated change already part of dependabot.

Reply:
But you haven't merged it, yet? So as long as trunk doesn't contain it, it is a valid change.